src/bin/pg_autoctl/monitor.c

```diff
@@ -2554,8 +2609,24 @@ monitor_start_maintenance(Monitor *monitor, int nodeId)
 								   paramCount, paramTypes, paramValues,
 								   &context, &parseSingleValueResult))
 	{
+		if (monitor_retryable_error(context.sqlstate) &&
+			!pgsql_retry_policy_expired(retryPolicy))
+		{
+			int sleep = pgsql_compute_connection_retry_sleep_time(retryPolicy);
```

Comment:
Let's do this everywhere. People expect `sleep` to be a function, even github syntax highlighting does that it seems.

```suggestion
			int sleepTime = pgsql_compute_connection_retry_sleep_time(retryPolicy);
```

Reply:
Done now.